.travis.yml

```diff
@@ -10,9 +10,11 @@ addons:
     - libssl-dev
     - libunwind8
     - zlib1g
-env:
-  - KOREBUILD_DNU_RESTORE_CORECLR=true KOREBUILD_TEST_DNXCORE=true
 mono:
   - 4.0.5
+os:
+  - linux
+  - osx
+osx_image: xcode7.1
 script:
-  - ./build.sh --quiet verify
+  - ./build.sh verify
```

Comment:
Now that you mentioned it, I think I might have removed to much. `dotnet` doesn't support it but `npm` and other tools still do. Let me see what I can do